src/requirements/data/majors/ess.ts

```diff
@@ -0,0 +1,641 @@
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Economics',
+    description: 'Choose one course. ',
+    source: 'https://cals.cornell.edu/environment-sustainability/education/core-curriculum',
+    checker: includesWithSingleRequirement('AEM 1500', 'AEM 2500'),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Field/Engaged Experience',
+    description: 'Choose one course. ',
+    source: 'https://cals.cornell.edu/environment-sustainability/education/core-curriculum',
+    checker: includesWithSingleRequirement(
+      'BIOEE 3611',
+      'ENTOM 2120',
+      'NTRES 2100',
+      'NTRES 2400',
+      'NTRES 3260',
+      'NTRES 4560',
+      'BIOSM 2500'
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Sustainability Science Colloquium',
+    description: 'Choose one course. ',
+    source: 'https://cals.cornell.edu/environment-sustainability/education/core-curriculum',
+    checker: includesWithSingleRequirement('ENVS 2000', 'BEE 2000', 'BEE 2010'),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Capstone Course',
+    description: 'Choose one course. ',
+    source: 'https://cals.cornell.edu/environment-sustainability/education/core-curriculum',
+    checker: includesWithSingleRequirement(
+      'EAS 4443',
+      'ENVS 4444',
+      'ENVS 4940',
+      'NTRES 4600',
+      'BIOSM 3750'
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Concentration',
+    description:
+      'All students must select one of six concentrations, consisting of six to ten additional courses beyond the core. ' +
+      'CoursePlan does not support the Student-Designed (SD) concentration. ',
+    source: 'https://cals.cornell.edu/environment-sustainability/education/concentrations',
+    fulfilledBy: 'toggleable',
+    fulfillmentOptions: {
+      'Environmental Economics': {
+        description:
+          'A. Required Courses, ' +
+          'B. Resource/Environmental Economics (choose two), ' +
+          'C. Data Analysis/Econometrics (choose one)',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['AEM 2500'],
+          ['MATH 1110'],
+          ['ECON 1110'],
+          ['ECON 1120'],
+          ['ECON 3030'],
+
+          ['AEM 4500', 'AEM 4510', 'AEM 4515', 'ECON 3850'],
+          [
+            'AEM 3100',
+            'AEM 4110',
+            'ECON 3120',
+            'ECON 3140',
+            'ILRST 2110',
+            'PLSCS 2200',
+            'STSCI 4060',
+          ]
+        ),
+        perSlotMinCount: [1, 1, 1, 1, 1, 2, 1],
+        slotNames: [
+          'Course',
+          'Course',
+          'Course',
+          'Course',
+          'Course',
+          'Resource/Environmental Economics',
+          'Data Analysis/Econometrics',
+        ],
```

Comment:
Nit: To be consistent with the slotNames of the other concentrations, should we include the group names e.g. 'A. Required Course', 'B. Resource/Environmental Economics', 'C. Data Analysis/Econometrics'?

Reply:
yes, added these!